File: python/Algorithm6.py
# Algorithm 6: Find the Fibonacci series till the term less than 1000

first_term = 0
second_term = 1
print(first_term, second_term)

while second_term < 1000:
    temp = second_term
    second_term = second_term + first_term
    first_term = temp
    if second_term < 1000:
        print(second_term)

File: javascript/Algorithm6.js
// Algorithm 6: Find the Fibonacci series till the term less than 1000

let firstTerm = 0;
let secondTerm = 1;

console.log(firstTerm + " " + secondTerm);

while (secondTerm < 1000) {
  let temp = secondTerm;
  secondTerm = secondTerm + firstTerm;
  firstTerm = temp;
  if (secondTerm < 1000) {
    console.log(secondTerm);
  }
}
